osdep/io.c

```diff
@@ -692,6 +692,58 @@ off_t mp_lseek(int fd, off_t offset, int whence)
     return _lseeki64(fd, offset, whence);
 }
 
+void *mp_dlopen(const char *filename, int flag)
```

Comment:
Not sure if any of the flags could/should be translated to winapi equivalents, CC @rossy.

Reply:
I found this dlfcn implementation for win32: https://github.com/dlfcn-win32
They try to translate some of the RTLD_ flags, e.g. RTLD_LOCAL: https://github.com/dlfcn-win32/dlfcn-win32/blob/master/dlfcn.c#L286
by keeping a doubly linked list of opened libraries handles.
Given that we don't generally dlclose opened libraries, I guess the behavior should be the same.

As far as I understand, RTLD_NOW is the default behavior on windows.
There's a way to not make it resolve symbols at all (LoadLibraryEx + DONT_RESOLVE_DLL_REFERENCES flag) which is not the same as RTLD_LAZY (anyway, its use it's discouraged by the official documentation in favor of other flags for other purposes).